packages/oo-ascii-tree/README.md

```diff
@@ -0,0 +1,166 @@
+# oo-ascii-tree
+
+Renders ASCII trees from an object-oriented object graph.
```

Comment:
Is there anything really OO graph-specific here? I feel like this is underselling it to a niche when it's actually able to be a general purpose "make me a tree view of this"

Reply:
It's just that there's already an **ascii-tree** npm module and this one is different in the sense that it uses classes to represent the nodes (the other one uses a string syntax as input). Nothing interesting.